es6-promises/index.js

```diff
@@ -0,0 +1,11 @@
+const takeAChance = require('./take-a-chance');
+
+const promise1 = takeAChance('Dulce');
+
+promise1.then(value => {
+  console.log(value);
+});
+
+promise1.catch(error => {
+  console.log(error.message);
+});
```

Comment:
This will work, but traditionally we chain these together. For example:
```JAVASCRIPT
promise1.then(value => {
  console.log(value);
}).catch(error => {
  console.log(error.message);
});
```